code/modules/mob/living/living.dm

```diff
@@ -505,6 +505,9 @@ Sorry Giacom. Please don't be mad :(
 	heal_overall_damage(1000, 1000)
 	ExtinguishMob()
 	fire_stacks = 0
+	if(ishuman(src))
+		var/mob/living/carbon/human/H = src
+		H.modify_drunkenness(-H.drunkenness)

```

Comment:
why not put this here:https://github.com/tgstation/-tg-station/blob/6949167e6fc674e71988be6713925d8953bb1e2d/code/modules/mob/living/carbon/human/human.dm#L948